### PR TITLE
che #6306: Using 'OpenShiftConfigBuilder' in 'OpenShiftClientFactory'

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Using 'OpenShiftConfigBuilder' in 'OpenShiftClientFactory'

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/6306

### CODE REVIEW

File: infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftClientFactory.java [hunk 1]
```diff
@@ -12,42 +12,42 @@
...
 
     if (doTrustCerts != null) {
-      config.setTrustCerts(doTrustCerts);
+      configBuilder.withTrustCerts(doTrustCerts);
     }
   }
 
   public OpenShiftClient create() {
-    return new DefaultOpenShiftClient(config);
+    return new DefaultOpenShiftClient(configBuilder.build());
```
Comment (sleshchenko) [inline infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftClientFactory.java]: I'd try to build config once in the constructor, save it in a field and use here.

**Review by sleshchenko — APPROVED** [review]
OK for me. But please take a look my comment.

File: infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftClientFactory.java (hunk 1 above)
Comment (ibuziuk) [reply infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftClientFactory.java]: good point

**Comment by ibuziuk** [comment]
ci-build

**Review by akorneta — APPROVED** [review]

**Pull request merged**